core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -50,8 +56,16 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
   override def getWorkspace(workspaceId: UUID, ctx: RawlsRequestContext): WorkspaceDescription =
     getWorkspaceApi(ctx).getWorkspace(workspaceId, null) // use default value for role
 
-  override def createWorkspace(workspaceId: UUID, ctx: RawlsRequestContext): CreatedWorkspace =
-    getWorkspaceApi(ctx).createWorkspace(new CreateWorkspaceRequestBody().id(workspaceId))
+  override def createWorkspace(workspaceId: UUID,
+                               workspaceType: WorkspaceType,
+                               ctx: RawlsRequestContext
+  ): CreatedWorkspace = {
+    val stage = workspaceType match {
+      case WorkspaceType.RawlsWorkspace => WorkspaceStageModel.RAWLS_WORKSPACE
+      case WorkspaceType.McWorkspace    => WorkspaceStageModel.MC_WORKSPACE
+    }
+    getWorkspaceApi(ctx).createWorkspace(new CreateWorkspaceRequestBody().id(workspaceId).stage(stage))
```

Comment:
My first reaction: how did this ever work for MC_WORKSPACEs previously????

Looking deeper at it, `MultiCloudWorkspaceService` never calls this `createWorkspace`; it calls `createWorkspaceWithSpendProfile` instead. I wonder if these two methods are worth combining. That's out of scope for this PR (we could yak-shave forever).

Reply:
Agreed to punt this....